solid-b4c-rods/core.ini

```diff
@@ -0,0 +1,259 @@
+% ------ 333 rods per quarter, SVF=0.907268 (see http://localhost:8888/notebooks/Desktop/git/ta_xe_equilibrium_test/SVF.ipynb))
+
+surf 11 inf
+surf 12 inf
+surf 13 inf
+surf 14 inf
+surf 15 inf
+surf 16 inf
+
+cell 100 2 fuel -11
+cell 101 3 zrh -12
+cell 102 4 sic  -13
+cell 103 5 ctrlPois  -14
+cell 104 6 air -15
+cell 105 7 hast -16
+
+
+surf 1 cyl    0.0 0.0 1.15
+surf 2 cyl    0.0 0.0 1.25
+surf 3 sqc    0.0 0.0 1.5
+
+surf 4 pz -150                %Top of Active Core
+surf 5 pz  150                %Bottom of Active Core
+
+surf 6 pz -175                %Top of core
+surf 7 pz  175                %Bottom of core
+
+surf 8 pz -180	      	      %Top of the vessel
+surf 9 pz 180	        	      %Bottom of the vessel
+
+cell 1  M  fill 3 -1   -5 4
+cell 2  M  fill 4 -2 1 -5 4
+cell 3  M  fill 2 -3 2 -5 4
+cell 4  M  fill 2  -3  -4 6
+cell 5  M  fill 2  -3  -7 5
+
+cell 6  F  fill 2  -3  -7 6
+cell 7  F  fill 7  -3  -9 7
+cell 8  F  fill 7  -3  -6 8
+
+%The CR is said to be in a voided thimble with similar design to MSRE
+%Assumed thickness/outer radius was constant for each material
+%Assumed GT took up  1 moderator rod pitch
+
+surf 120 cyl 0.0 0.0 1.5        %OR for  CR GT
+surf 121 cyl 0.0 0.0 1.425      %IR for  CR GT
+
+surf 122 cyl 0.0 0.0 0.855      %OR for CR clad
+surf 123 cyl 0.0 0.0 0.810      %OR for CR pois
+%surf 124 cyl 0.0 0.0 0.630     %OR for CR clad
```

Comment:
I have been "yelled" at before for leaving commented code, you should just delete the lines

Reply:
Remove all commented lines, thanks.